tests/unit/test_winAPI/test_displayTracking.py

```diff
@@ -5,77 +5,197 @@
 
 import unittest
 
-from winAPI import displayTracking
 from winAPI.displayTracking import (
-	_updateOrientationState,
+	_getNewOrientationState,
 	Orientation,
 	OrientationState,
 )
 
 
 class Test_UpdateOrientationState(unittest.TestCase):
-	def setUp(self) -> None:
-		displayTracking._orientationState = OrientationState()
-
-	def tearDown(self) -> None:
-		displayTracking._orientationState = OrientationState()
 
 	def test_landscapeInitiallySet(self):
-		monitorSet = _updateOrientationState(1, 2)
-		self.assertTrue(monitorSet)
+		newStyle = _getNewOrientationState(
+			previousState=OrientationState(),  # simulates initialization

```

Comment:
>This potentially sounds like a bug, should NVDA set the initial state to accurately represent the orientation / size?

I believe the main reason we don't do this is because it would require a different function to initially look up the orientation state, as we currently only determine it via the state change events.

The only case where this would be a bug is if the first display size change event is a monitor change without an orientation state change.

I think we should initialize the state to begin with like the power state..